src/Footer.css

```diff
@@ -0,0 +1,5 @@
+.footer {
+    padding: 5%;
```

Comment:
note: percentage paddings are generally not the best idea because it's based on screen size, which could be a phone, or a very big monitor. I'd stick to static padding